be/src/service/starrocks_main.cpp

```diff
@@ -207,7 +207,6 @@ int main(int argc, char** argv) {
     }
 #endif
 
-#ifdef WITH_BLOCK_CACHE
     if (starrocks::config::block_cache_enable) {
```

Comment:
Do you think we can turn on block cache by default since we can change the quota online?
- we can set the default disk path as the same as storage engine 
- we can set the default disk quota and mem quota as 0 but we can modify quota by SQL
- make sure the 0 quota won't affect the performance